projects/goharbor/harbor/Makefile

```diff
@@ -0,0 +1,41 @@
+BASE_DIRECTORY=$(shell git rev-parse --show-toplevel)
+GIT_TAG?=$(shell cat GIT_TAG)
+GOLANG_VERSION?="1.17"
+
+REPO=harbor
+REPO_OWNER=goharbor
+REPO_SUBPATH=./src
+
+BASE_IMAGE_NAME?=eks-distro-base
+DOCKERFILE_FOLDER=./docker/linux/$(IMAGE_NAME)
+
+FETCH_BINARIES_TARGETS=eksa/helm/chartmuseum
+
+IMAGE_NAMES=harbor-chartserver
+
+HARBOR_CHARTSERVER_IMAGE_COMPONENT=harbor/harbor-chartserver
+
+$(call IMAGE_TARGETS_FOR_NAME, harbor-chartserver): harbor-chartserver-useradd/images/export
+
+harbor-chartserver-useradd/images/export: IMAGE_USERADD_USER_NAME=chart IMAGE_USERADD_USER_ID=10000
+
+PRECOMPILE_PREPARE=swagger_prepare
+
+DOCKERFILEPATH=$(PROJECT_ROOT)/$(REPO)/make/photon
```

Comment:
Is this used?

Reply:
it is used in DOCKERFILEPATH_CHART_SERVER, but yeah I can fold this in.